src/state_calc_clean.py

```
@@ -3,8 +3,10 @@
 from copy import deepcopy
 from random import choice, choices, shuffle
 
-from all_inputs import InputTemplate
-from log_module import logger
+import numpy as np
```

Ruff (0.11.9)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)